packages/text-emoji/src/components/emoji-picker/index.tsx

```diff
@@ -1,24 +1,27 @@
 /**
- * Currently, there are no type definitions in @emoji-mart/react, so we implement our own.
- * ref. https://github.com/missive/emoji-mart/blob/665a58031c7b5b753c06e8581dcfe51741098275/packages/emoji-mart-react/react.js
+ * Currently, type definitions in @emoji-mart/react is't perfect, so we implement our own.
+ * ref. https://github.com/missive/emoji-mart/issues/576
  */
 import { useRef, useEffect } from 'react';
-import { Data, Picker, PickerProps, BaseEmoji } from 'emoji-mart';
+import { Picker, PickerProps, Emoji as EmojiBaseType } from 'emoji-mart';
 import { styles } from './index.css';
 import i18n from '@emoji-mart/data/i18n/ja.json';
 import { themeVars } from '@bento-editor/core';
 
-export type Emoji = Data['emojis'][number] & BaseEmoji;
+export type Emoji = typeof EmojiBaseType.Props;
 
 declare module 'emoji-mart' {
-  export class Picker {
-    constructor(
-      _params: PickerProps & {
-        data: Data;
-        ref?: React.RefObject<HTMLElement>;
-      }
-    );
-    update(props: PickerProps): void;
+  interface Category {
+    id: string;
+    name: string;
+    emojis: string[];
+  }
+
+  interface Data {
+    compressed: boolean;
+    categories: Category[];
+    emojis: { [key: string]: Emoji };
```

Comment:
本当にただの書き方の問題なんですが、bentoだとこういう型はおそらく `Record` で書いているので揃えたいかもです....(他にこういう書き方している箇所見て書いたのであればスルーで大丈夫です)
```suggestion
    emojis: Record<string, Emoji>;
```

Reply:
すまん。元々@types/emoji-martに実装されていた型を持ってきていたのでtsumikiに合わせる

Reply:
了解です